components/modals/FilterModal.tsx

```diff
@@ -1,15 +1,270 @@
+                  label="Room 2"
+                  onPress={() => setSelectedRoom('Room 2')}
+                  selected={selectedRoom === 'Room 2'}
+                  isCenter
+                  style={{ borderColor: colors.text }}
+                />
+                <FilterButton
+                  label="Room 3"
+                  onPress={() => setSelectedRoom('Room 3')}
+                  selected={selectedRoom === 'Room 3'}
+                  style={styles.right}
+                />
+              </View>
+
+              <Space size={20} />
+
+              <StyledText size="lg" font="bold">
+                Session Type
+              </StyledText>
+
+              <Space size={8} />
+
+              <View style={[styles.rowGroup, { borderColor: colors.text }]}>
+                <FilterButton
+                  label="Keynote"
+                  onPress={() => setSelectedSessionType('Keynote')}
+                  selected={selectedSessionType === 'Keynote'}
+                  style={styles.left}
+                />
+                <FilterButton
+                  label="Codelab"
+                  onPress={() => setSelectedSessionType('Codelab')}
+                  selected={selectedSessionType === 'Codelab'}
+                  isCenter
+                  style={{ borderColor: colors.text }}
+                />
+                <FilterButton
+                  label="Session"
+                  onPress={() => setSelectedSessionType('Session')}
+                  selected={selectedSessionType === 'Session'}
+                  style={styles.right}
+                />
+              </View>
+              <Space size={12} />
+              <View style={[styles.rowGroup, { borderColor: colors.text }]}>
+                <FilterButton
+                  label="Lightining talk"
+                  onPress={() => setSelectedSessionType('Lightining talk')}
+                  selected={selectedSessionType === 'Lightining talk'}
+                  style={styles.left}
+                />
+                <FilterButton
+                  label="Panel Discussion"
+                  onPress={() => setSelectedSessionType('Panel Discussion')}
+                  selected={selectedSessionType === 'Panel Discussion'}
+                  style={[styles.withborder, { borderColor: colors.text }]}
+                />
+              </View>
+
+              <Space size={30} />
+
+              <PrimaryButton label="Filter" onPress={handleFilter} />
+
+              <Space size={20} />
+            </View>
+          </SafeAreaView>
+        </View>
+      </Modal>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  centeredView: {
+    flex: 1,
+    alignItems: 'center',
+    backgroundColor: 'rgba(0, 0, 0, 0.8)',
+  },
+  header: {
+    width: '100%',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    paddingVertical: 8,
+  },
+  modalView: {
+    width: '100%',
+    height: '90%',
```

Comment:
![Screenshot_1689147134](https://github.com/droidconKE/droidconKE2023ReactNative/assets/28436092/4ff5ca43-ea13-45bf-a038-d2120087ccd4)

On emulator there's no space below FILTER button. I think it's caused by this height property

Reply:
should be fixed with new padding values, please check